distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
 // -w800 -h600 +a0.3
 
 #version 3.7;
-
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
```

Comment:
As I'm saying, we can't perfectly match the 3.6 results, even when it comes to "simple" stuff like ambient and diffuse: As soon as they combine with any other effect (or even with each other), the simple power-law relationship comes crashing down. So rather than use precisely calculated values that only match under hypothetical circumstances, I'd advocate nicely rounded values.